site/build-and-publish.sh

```diff
@@ -13,6 +13,10 @@ if [ -n "${TRAVIS_TAG}" ]; then
   OUTPUT="weave/${TRAVIS_TAG}"
 fi
 
+if [ "${BRANCH}" = "latest_release_doc_updates" ]; then
```

Comment:
hmm. good point. What happens when we make a new release? won't this branch go stale? Perhaps the release script should include a merge from the release tag to this branch. or s.t. like that.